crates/filtering/src/hysteresis.rs

```diff
@@ -27,3 +27,29 @@ pub fn less_than_with_hysteresis(
                 -hysteresis
             }
 }
+
+pub fn greater_than_with_hysteresis_from_tresholds(
```

Comment:
What do you think about only wrapping the other functions. That is, the new ones call the old version or vice versa...

Reply:
Done